ui/ui-invoices/package.json

```diff
@@ -14,16 +14,14 @@
     "@k-ramel/react": "^0.12.0",
     "babel-polyfill": "^6.26.0",
     "date-fns": "^1.29.0",
-    "hoc-little-router": "^1.2.0",
-    "hoc-react-loader": "^6.3.0",
     "k-ramel": "^0.12.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-form": "^3.0.1",
     "react-scripts": "1.1.4",
-    "recompose": "^0.26.0",
     "redux-little-router": "^15.0.0",
-    "tinycolor2": "^1.4.1"
+    "tinycolor2": "^1.4.1",
```

Comment:
On est obligé de l'importer vu que tu utilise pas le loadingIndicator ?

Reply:
We are in a public repository, please try to speak english.

I think we can, but we have to fix how `hoc-react-loader` works for this dependency first.
So not for now.